node/src/components/contract_runtime.rs

```diff
@@ -620,9 +649,17 @@ impl ContractRuntime {
             let queue = &mut *exec_queue.lock().expect("mutex poisoned");
             queue.remove(&new_execution_pre_state.next_block_height)
         };
+
+        let activation_point_block_height = effect_builder.get_activation_point_era_id().await;
```

Comment:
I think we only need to make this call if the `activation_point_block_height` passed into the function was `None`.

If we can change to not have optional values for these though, we don't need this call - we just copy what was passed in.

Reply:
Correct, removed unnecessary call